plugin.yaml

```diff
@@ -144,13 +248,25 @@ node_types:
         properties:
             device_name:
                 default: ''
+                description: (unused)
```

Comment:
Is this true? I found no mention of this in the code or in any docs for the plugin.